browser/src/app/IconUtil.ts

```diff
@@ -0,0 +1,134 @@
+// @ts-strict-ignore
```

Comment:
`@ts-strict-ignore` should be used only in old files (which require some changes).

please remove it